onnx2kerastl/operation_layers.py

```diff
@@ -773,12 +773,16 @@ def convert_nms(node, params, layers, lambda_func, node_name, keras_name):
             score_threshold = score_threshold[0]
     num_classes = scores.shape[1]
     all_results = []
+    try:
+        iou_threshold = iou_threshold[0]
+    except IndexError: #iou threshold is already a scalar
+        pass
```

Comment:
We are using `ensure_float` on `score_threshold` for the same reason. why not using here as well?

Reply:
What if the IOU_THRESHOLD is dynamically computed? i.e. a KerasTensor and not an Eager tensor? Currently, ensure_float will not deal properly with that IMO..

Suggest we hold this off until we make some improvement to ensure_float and test it and then maybe standartize it across Onnx2kerras